lib/beaker/options/subcommand_options_file_parser.rb

```diff
@@ -3,18 +3,30 @@ module Options
     #A set of functions to read options files
     module SubcommandOptionsParser
 
+      HOMEDIR_OPTIONS_FILE_PATH = ENV['HOME']+'/.beaker/subcommand_options.yaml'
+
+      def self.parse_options_file(options_file_path)
+        result = OptionsHash.new
+        if File.exist?(options_file_path)
+          result = YAML.load_file(options_file_path)
+        end
+        result
+      end
+
       # @return [OptionsHash, Hash] returns an empty OptionHash or loads subcommand options yaml
       #   from disk
-      def self.parse_subcommand_options(argv)
+      def self.parse_subcommand_options(argv, home_dir=false)
```

Comment:
I'm confused as to how this (passing a `home_dir` boolean argument) is better than having the user specify the path to the file & using the constant in the call if you'd like to use the home directory.

Or even better, we could use the default behavior to not need to specify the home directory (although I'm not sure if the home directory or the current directory would be a better default).